### PR TITLE
Add T-shirt size to mentors

### PR DESCRIPTION
**Pull request by pedrofp4444**
Feature to add new field which allows to safe the t-shirt size of the mentors.

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -11,5 +11,3 @@ services:
       POSTGRES_HOST: ${DB_HOST:-localhost}
     volumes:
       - /var/lib/postgresql/data
-    ports:
-      - "5432:5432"
```
Comment (ruilopesm) [inline docker-compose.dev.yml]: Can you remove this change from this PR? 🙏🏼

**Review by ruilopesm — DISMISSED** [review]

File: docker-compose.dev.yml (hunk 1 above)
Comment (pedrofp4444) [reply docker-compose.dev.yml]: Ohh of course! I totally forgot about it...

**Comment by ruioliveira02** [comment]
Watch your branch names. Our convention is ```<your initials>/<what you are doing>```. You are missing the second part here

File: priv/repo/migrations/20221129193311_alter_mentors.exs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+defmodule Bokken.Repo.Migrations.AlterMentors do
+  use Ecto.Migration
+
+  def change do
+    alter table(:mentors, primary_key: false) do
+      add :t_shirt, :string
```
Comment (ruioliveira02) [inline priv/repo/migrations/20221129193311_alter_mentors.exs]:
```suggestion
      add :t_shirt, :citext
```
This way the value is case insensitive. It shouldn't really matter, but I think it makes more sense.
(Citext is a Postgresql extension)

**Review by ruioliveira02 — APPROVED** [review]

**Pull request merged**